test/closure/subclassing_closure.wren

```diff
@@ -0,0 +1,16 @@
+var f = null
+
+{
+  var a = "a"
+  f = new Fn {
+    IO.print(a)
+  }
+}
+//@FIXME: repair the segfault create a propper test for the subclassing
+
+//var closureSegfault = f
+//class SubClosureSegfault is closureSegfault {} 
```

Comment:
This is running into another bug in the VM. In this case, the superclass isn't the closure *class* it's the closure *instance*. There's another issue (with a TODO) where we need to ensure the superclass is a class at all. It's not checking that, hence the nasty segfault.